File: Java/Maths/Fibonacci Series to nth terms.java
import java.util.Scanner;
class fibonacci_nthTerms {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter a positive number: ");
        int num = input.nextInt();

        int term1 = 0, term2 = 1;
        int nextTerm = term1 + term2;
        if(num <= 0){
            System.out.println("Error! Entered number must be a positive number.");
        }else{
            System.out.println("\nFibonacci series to " + num + "th term(s):");
            while (nextTerm <= num) {
                nextTerm = term1 + term2;
                term1 = term2;
                term2 = nextTerm;
                System.out.print(term1 + ", ");
            }
        }
    }
}

File: Python/Maths/Fibonacci Series to nth terms.py
num = int(input("Enter a positive number: "))
term1 = 0; term2 = 1
nextTerm = term1 + term2
if num <= 0:
    print("Error! Entered number must be a positive number.")
else:
    print("\nFibonacci series to", str(num)+"th term(s):")
    while (nextTerm <= num):
        nextTerm = term1 + term2
        term1 = term2
        term2 = nextTerm
        print(term1, end=", ")